miqa/core/management/commands/populate.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth.models import User
+import djclick as click
+
+from miqa.core.rest.session import import_data
+from miqa.core.tests.factories import SessionFactory, UserFactory
+
+
+@click.command()
+@click.option('--csv', type=click.Path(exists=True))
+@click.option('--username', type=click.STRING, help='username for session creator')
+def command(csv, username):
+
+    if username:
+
+        user = User.objects.get(username=username)
+
+    else:
+
+        user = UserFactory()
```

Comment:
Let's make this use `User.objects.first()` if no username is passed. That will mean, in practice, that it belongs to the superuser created by the developer initially.